model/client.go

```diff
@@ -33,22 +35,37 @@ type Result struct {
 }
 
 type Client struct {
-	Url        string       // The location of the server like "http://localhost/api/v1"
+	Url        string       // The location of the server like "http://localhost:8065"
+	ApiUrl     string       // The api location of the server like "http://localhost:8065/api/v1"
 	HttpClient *http.Client // The http client
 	AuthToken  string
+	AuthType   string
 }
 
 // NewClient constructs a new client with convienence methods for talking to
 // the server.
-func NewClient(url string) *Client {
-	return &Client{url, &http.Client{}, ""}
+func NewClient(url, apiUrl string) *Client {
```

Comment:
we should figure out a better way todo this.   client.go will be our driver library that we distribute.  having 2 urls is confusing.